clients/python/girder_client/cli.py

```diff
@@ -146,6 +146,7 @@ def format_options(self, ctx, formatter):
                    '[default: %s]' % GirderClient.DEFAULT_API_ROOT,
               show_default=True,
               cls=_AdvancedOption)
+@click.version_option()
```

Comment:
We should probably pass the version explicitly to avoid the auto-discovery logic.

```
from girder_client import __version__

@click.version_option(version=__version__, prog_name='Girder command line interface')
```